Formula/otf2.rb

```diff
@@ -0,0 +1,29 @@
+class Otf2 < Formula
+  desc "Open Trace Format 2 file handling library"
+  homepage "https://www.vi-hps.org/projects/score-p/"
+  url "https://www.vi-hps.org/cms/upload/packages/otf2/otf2-2.1.1.tar.gz"
+  sha256 "01591b42e76f396869ffc84672f4eaa90ee8ec2a8939755d9c0b5b8ecdcf47d3"
+
+  depends_on "python"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--enable-backend-test-runs"
```

Comment:
What's the impact of this on the resulting binary?

Reply:
As far as I understand, this allows one to run some testing on the library built using make check.

Reply:
Not sure if it'd be good to have such debug options in the build we ship out to users.

Reply:
Ok I'll remove and push

Reply:
done